### ISSUE TITLE
Get rid of latex

### ISSUE THREAD

**Issue by viroulep** [suushiemaniac/wca-edudoc#1]
The paragraph in [the readme](https://github.com/suushiemaniac/wca-edudoc#why-the-heck-use-latex) is not accurate, as `wkhtmltopdf` let the user specify the `--header-html` and `--footer-html` options.
We actually got rid of latex for generating the pdf for the WCA Regulations, because of the extreme pain of getting texlive installed when spinning up an image of whatever component was building the regulations (ie: on the website's server or developers computer, but the same reasoning could apply to a travis instance that would build the tutorial document).

The WCA Regulations headers and footer are generated with `wkhtmltopdf`, and include information about the current section on the page and the page number (respectively).

You can check out how the Regulations compiler craft the `wkhtmltopdf` command [here](https://github.com/thewca/wca-regulations-compiler/blob/master/wrc/wrc.py#L85-L113), header and footer are in the [`data`](https://github.com/thewca/wca-regulations-compiler/tree/master/wrc/data) directory, they contain the logic related to getting section information and page numbering.

With an extension of the custom css you're already using it should should be rather easy to handle the only real tricky part with `wkhtmltopdf` (page breaks), if it's even needed at all (for the WCA Regulations it was tricky because we had `ul`s spanning over multiple pages).

If I understand correctly the motivation behind using latex, it should be quite straightforward to generate a similar result using just `wkhtmltopdf` :)

**Comment by leonopulos** [comment]
This is done! See [PR](https://github.com/suushiemaniac/wca-edudoc/pull/2) above.

Please open a new issue if something else needs to happen before we can integrate this into thewca.

**Comment by gregorbg** [comment]
This was solved by Leon in #2

**Comment by viroulep** [comment]
That's awesome, thanks @leonopulos!
